src/macaron/config/defaults.py

```diff
@@ -37,24 +37,26 @@ def get_list(
         If ``strip`` is True  (default: True), strings are whitespace-stripped and empty strings
         are removed from the final result.
 
-        If ``duplicated_ok`` is True (default: False), duplicated values are not removed from the final list.
+        If `duplicated_ok` is False, duplicated elements which come after the their first instances will
+        be removed from the list. This operation happens after ``strip`` is handled.
 
+        The order of non-empty elements in the list is preserved.
         The content of each string in the list is not validated and should be handled separately.
 
         Parameters
         ----------
         section : str
             The section in ``defaults.ini``.
-        item : str
-            The item to parse the list.
+        option : str
+            The option whose value will be split into the return list of strings.
         delimiter : str | None
             The delimiter used to split the strings.
         fallback : list | None
             The fallback value in case of errors.
-        duplicated_ok : bool
-            If True allow duplicate values.
-        strip: bool
+        strip : bool
             If True, strings are whitespace-stripped and any empty strings are removed.
+        duplicated_ok : bool
+            If True, duplicated elements are removed from the list.
```

Comment:
The description should be 
```
         If True, the duplicated elements will not be removed from the list.
```
?
I think this argument name is confusing, so maybe change it to

```suggestion
        remove_duplicates : bool
            If True, duplicated elements will be removed from the list.
```

Reply:
Modified in https://github.com/oracle/macaron/pull/660/commits/3b7cd97fd7b245ad8f05994aad7f4e3ff38f8427